src/assets/scripts/airport/Arrival/ArrivalBase.js

```diff
@@ -360,13 +360,14 @@ export default class ArrivalBase {
                     // if point before next fix
                     const nextFix = waypoint;
                     const previousFix = waypointModelList[j - 1];
-                    const heading = bearing(previousFix.gps, nextFix.gps);
-                    const calculatedRadialDistance = fixRadialDist(previousFix.gps, heading, spawnOffset);
+                    const heading = bearingToPoint(previousFix.gpsXY, nextFix.gpsXY);
+                    const spawnPoint = fixRadialDist(previousFix.gps, heading, spawnOffset);
+                    const spawnPosition = new PositionModel(spawnPoint, this.airport.position, this.airport.magnetic_north);
```

Comment:
something in this area broke initial spawn positions such that the initially spawned aircraft are not visible.  They are created, but never show up in the view.